src/App.css

```diff
@@ -1,32 +1,3 @@
 .App {
```

Comment:
- [ ] Kindly write a descriptive README following the Microverse [template](https://github.com/microverseinc/readme-template).

Reply:
- [ ] All boilerplate files and icons should be deleted as they are not required for the project. e.g In the public folder(the logo images, manifest.json and robots.txt).

Reply:
Hi @ZeenatLawal,
But I had already fixed those issues, Kindly re-review. Thank you!

Reply:
Hello @billionsjoel, kindly submit for a re-review on your dashboard. Thanks.